projects/tests.py

```diff
@@ -750,16 +661,103 @@ def test_ociso_component_not_returned(self):
             format="json",
         )
         self.assertEqual(resp.status_code, 200)
-        formatedResponse = json.loads(resp.content)
-        for test in formatedResponse:
-            self.assertNotEqual(test.get("title"), "OCISO")
+
+        content = resp.json()
+        for test in content:
+            with self.subTest():
+                self.assertNotEqual(test.get("title"), "OCISO")
 
     def test_private_component_not_returned(self):
         resp = self.client.get(
             "/api/projects/" + str(self.test_project.id) + "/components-not-added/",
             format="json",
         )
         self.assertEqual(resp.status_code, 200)
-        formatedResponse = json.loads(resp.content)
-        for test in formatedResponse:
-            self.assertNotEqual(test.get("title"), "private component")
+        content = resp.json()
+
+        for test in content:
+            with self.subTest():
+                self.assertNotEqual(test.get("title"), "private component")
+
+
+class RetrieveUpdateProjectControlViewTestCase(AuthenticatedAPITestCase):
+    @classmethod
+    def setUpTestData(cls):
+        user = User.objects.create()
+        token = Token.objects.create(user=user)
+
+        cls.user, cls.token = user, token
+
+        call_command(
+            "load_catalog",
+            name="NIST Test Catalog",
+            catalog_file="blueprintapi/testdata/NIST_SP-800-53_rev5_test.json",
+            catalog_version="NIST 800-53",
+            impact_level=Catalog.ImpactLevel.LOW,
+        )
+
+        test_catalog = Catalog.objects.get(name="NIST Test Catalog")
+
+        cls.project = Project.objects.create(
+            title="Test project",
+            acronym="TP",
+            catalog_version="NIST 800-53",
+            impact_level="low",
+            location="other",
+            creator=user,
+            catalog=test_catalog,
+        )
+
+    def setUp(self):
+        self.client.force_authenticate(user=self.user, token=self.token)
+
+    def test_get_project_control(self):
+        response = self.client.get(
```

Comment:
Test currently fails because the backend checks for permissions based on the queryset model (`ProjectControl`) rather than what we want (`Project`). I'm working on figuring out the best way to tackle this since it's fairly common in this project to access a list of things associated with a particular project.